core/channel/channel_factory_base.cpp

```diff
@@ -17,10 +17,14 @@
 #include <string>
 #include <unordered_map>
 
+#include "base/session_local.hpp"
+
 namespace husky {
 
 thread_local int ChannelFactoryBase::default_channel_id = 0;
 thread_local std::unordered_map<std::string, ChannelBase*> ChannelFactoryBase::channel_map;
```

Comment:
@kygx-legend I just noticed this >_<. I think there will be lots of modifications if I change to `std::unique_ptr`? So maybe we fix this in a new PR?